ndc-client/src/models.rs

```diff
@@ -42,6 +42,7 @@ pub struct LeafCapability {}
 pub struct Capabilities {
     pub query: QueryCapabilities,
     pub explain: Option<LeafCapability>,
+    pub mutation: Option<MutationCapabilities>,
```

Comment:
Right now there's no notion of not supporting mutations at all. You would simply not have any procedures. So this `Option` only exists for backwards compatibility, right? If so, I'd say let's just make a breaking change now.

Reply:
Alright. I removed the Option.